internal/services/v1/watch.go

```diff
@@ -157,3 +188,18 @@ func filterUpdates(objectTypes *mapz.Set[string], filters []datastore.Relationsh
 
 	return filtered
 }
+
+func convertWatchKindToContent(kinds []v1.WatchKind) datastore.WatchContent {
```

Comment:
this is correct but a small unit test would be good, just in case we change these types in the future

Reply:
Added! Though it feels hackish to make this function public just to be able to test it...

Reply:
If you add another test file within `package v1`, then you don't need to make it public. I'd do that instead